RELEASE_NOTES.md

```diff
@@ -4,6 +4,8 @@
 
 ### New features
 
+- Entities Filter and Table display
```

Comment:
- Table to list Entities (#111)
- Filter entities in table by servicePath, type and date (#112)

Reply:
this is not resolved, the issues are not referred in the RELEASE_NOTES